dcicutils/beanstalk_utils.py

```diff
@@ -1252,24 +1254,24 @@ def auth0_client_update(url):
         None
     """
     # generate a jwt to validate future requests
-    client = os.environ["Auth0Client"]
-    secret = os.environ["Auth0Secret"]
-    payload = {"grant_type": "client_credentials",
-               "client_id": client,
-               "client_secret": secret,
-               "audience": "https://hms-dbmi.auth0.com/api/v2/"}
-    headers = {'content-type': "application/json"}
-    res = requests.post("https://hms-dbmi.auth0.com/oauth/token",
+    client = os.environ['Auth0Client']
+    secret = os.environ['Auth0Secret']
+    payload = {'grant_type': 'client_credentials',
+               'client_id': client,
+               'client_secret': secret,
+               'audience': 'https://hms-dbmi.auth0.com/api/v2/'}
```

Comment:
Personally, I like `'...'` for tokens and `"..." for free format strings, so I would make URLs in doublequotes. But there's no set rule, so this is fine.